README.md

```diff
@@ -133,6 +133,67 @@ cd Binaries
 
 Please note the current build script is only tested on Ubuntu 14.04. Please update the script accordingly for other distributions.
 
+## Getting Started on OSX
+
+*Note that OSX is not officially supported yet, but it has been seen to work, YMMV. This build has been tested to work when the dependencies are installed via homebrew, YMMV if using FINK or MacPorts*
+
+Install dependecies with homebrew:
+
+```
+brew install libxml++ ossp-uuid openssl
+```
+
+As mentioned above, the Azure Storage Client Library for C++ depends on Casablanca.
+If you are using homebrew you can install it from there:
+```
+brew install cpprestsdk
+```
+
+Otherwise, you may need to build it. Follow [these instructions](https://github.com/Microsoft/cpprestsdk/wiki/How-to-build-for-Mac-OS-X) to compile it. Current version of the library depends on Casablanca version 2.8.0.
+
+Once this is complete, then:
+
+- Clone the project using git:
+```bash
+git clone https://github.com/Azure/azure-storage-cpp.git
+```
+The project is cloned to a folder called `azure-storage-cpp`. Always use the master branch, which contains the latest release.
+
+**Some notes about building**:
```

Comment:
How about use `brew link --force openssl`?  This command forcefully link all openssl libraries and headers to `/usr/local/`.  Any concern on this command?